app/assets/src/components/views/other/CliUserInstructions.jsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
```

Comment:
We have been putting simpler views directly in `components/views` (instead of other).